### PR TITLE
Remove jQuery from `user-notes.js`

### PR DESCRIPTION
**Pull request by cbravobernal**
## What

Remove jQuery from `user-notes.js`

## Why

jQuery is known to inflate the bundle size of packages, and in most cases, it can be replaced with native language functionality.

### CODE REVIEW

File: source/wp-content/themes/wporg-developer-2023/js/user-notes.js [hunk 1]
```diff
@@ -3,73 +3,79 @@
...
-	// Scroll to comment if comment date link is clicked
-	$( '#comments' ).on( 'click', '.comment-date', function( e ) {
-		// Scroll to comment and adjust for admin bar
-		// Add 16px for child comments
-		var pos = $( this ).offset();
-		$( 'html,body' ).animate( {
-			scrollTop: pos.top - wpAdminBar - 16
-		}, 1 );
+	// Scroll to comment if comment date link is clicked.
+	document.querySelectorAll( '#comments .comment-date' ).forEach( ( element ) => {
```
Comment (StevenDufresne) [inline source/wp-content/themes/wporg-developer-2023/js/user-notes.js]: I don't believe this code path is even working anymore as I can't find the `#comments` parent. Do I have that wrong?

 Should we remove this smooth scrolling if that is the case?

File: source/wp-content/themes/wporg-developer-2023/js/user-notes.js [hunk 2]
```diff
@@ -79,43 +85,40 @@
...
 	// Copied from code within core's wp-admin/js/common.js.
-	commentForm.bind('keydown.wpevent_InsertTab', function(e) {
-		var el = e.target, selStart, selEnd, val, scroll, sel;
+	commentForm.bind( 'keydown.wpevent_InsertTab', function ( e ) {
+		const elementTarget = e.target;
+		let scroll, sel;
 
-		if ( e.keyCode == 27 ) { // escape key
-			// when pressing Escape: Opera 12 and 27 blur form fields, IE 8 clears them
+		if ( e.key === 'Escape' ) {
```
Comment (StevenDufresne) [inline source/wp-content/themes/wporg-developer-2023/js/user-notes.js]: Can we remove this if statement if its for those very out-of-date browsers?

File: source/wp-content/themes/wporg-developer-2023/js/user-notes.js (hunk 1 above)
Comment (cbravobernal) [reply source/wp-content/themes/wporg-developer-2023/js/user-notes.js]: Yes, it was also not working here:
https://github.com/WordPress/wporg-developer/pull/532

Fixed in https://github.com/WordPress/wporg-developer/pull/530/commits/9bb5d46f03c8d726519f9acedc9d298e09592c75 and https://github.com/WordPress/wporg-developer/pull/530/commits/2d7c1ed6c931a42b5c3deaf895204ef65db39b92

Comment (cbravobernal) [reply source/wp-content/themes/wporg-developer-2023/js/user-notes.js]: Fixed in https://github.com/WordPress/wporg-developer/pull/530/commits/9bb5d46f03c8d726519f9acedc9d298e09592c75 and https://github.com/WordPress/wporg-developer/pull/530/commits/2d7c1ed6c931a42b5c3deaf895204ef65db39b92

**Comment by StevenDufresne** [comment]
Thanks for the PR!

**Pull request merged**